reference/filesystem/functions/file.xml

```diff
@@ -1,76 +1,205 @@
+         <listitem>
+          <simpara>
+           Cerca il file in <link 
+           linkend="ini.include-path">include_path</link>.
+          </simpara>
+         </listitem>
+        </varlistentry>
+        <varlistentry>
+         <term>
+          <constant>FILE_IGNORE_NEW_LINES</constant>
+         </term>
+         <listitem>
+          <simpara>
+           Ometterà newline alla fine di ogni elemento dell'array
+          </simpara>
+         </listitem>
+        </varlistentry>
+        <varlistentry>
+         <term>
+          <constant>FILE_SKIP_EMPTY_LINES</constant>
+         </term>
+         <listitem>
+          <simpara>
+           Salta le righe vuote
+          </simpara>
+         </listitem>
+        </varlistentry>
+        <varlistentry>
+         <term>
+          <constant>FILE_NO_DEFAULT_CONTEXT</constant>
+         </term>
+         <listitem>
+          <simpara>
+           Non usa il context di default
+          </simpara>
+         </listitem>
+        </varlistentry>
+       </variablelist>
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><parameter>context</parameter></term>
+     <listitem>
+       &note.context-support;
+     </listitem>
+    </varlistentry>
+   </variablelist>
+  </para>
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Restituisce il file in un array. Ogni elemento dell'array corrisponde a una
+   riga nel file, con newline incluso. In caso di errore,
+   <function>file</function> restituisce &false;.
+  </para>
+  <note>
+   <para>
+    Per ogni linea riportata nell'array risultante includerà newline, a meno
+    che non venga usato <constant>FILE_IGNORE_NEW_LINES</constant>.
+   </para>
+  </note>
+  &note.line-endings;
+ </refsect1>
+
+ <refsect1 role="errors">
+  &reftitle.errors;
+  <simpara>
+   A partire da PHP 8.3.0, genera un <exceptionname>ValueError</exceptionname>
+   se <parameter>flags</parameter> include valori non validi, come
+   <constant>FILE_APPEND</constant>.
+  </simpara>
+  <para>
+   Emette un <constant>E_WARNING</constant> a livello d'errore
+   se il file non esiste.
+  </para>
+ </refsect1>
+
+ <refsect1 role="changelog">
+  &reftitle.changelog;
+  <informaltable>
+   <tgroup cols="2">
+    <thead>
+     <row>
+      <entry>&Version;</entry>
+      <entry>&Description;</entry>
+     </row>
+    </thead>
+    <tbody>
+     <row>
+      <entry>8.3.0</entry>
+      <entry>
+       <exceptionname>ValueError</exceptionname> viene generato
+       per qualunque valore di <parameter>flags</parameter> non valido.
+      </entry>
+     </row>
+    </tbody>
+   </tgroup>
+  </informaltable>
+ </refsect1>
+
+ <refsect1 role="examples">
+  &reftitle.examples;
+  <para>
+   <example>
+    <title><function>file</function> esempio</title>
+    <programlisting role="php">
 <![CDATA[
 <?php
 // inserisce una pagina web in un array e la stampa. In questo esempio useremo il protocollo
```

Comment:
"Inserisce un file in un array." invece di "inserisce una pagina web in un array e la stampa."